lib/beacon_api/controllers/v2/beacon_controller.ex

```diff
@@ -61,14 +64,37 @@ defmodule BeaconApi.V2.BeaconController do
       version: "deneb",
       execution_optimistic: true,
       finalized: false,
-      data: %{
-        # TODO: return block as JSON
-        message: inspect(block)
-      }
+      data: to_json(block)
     })
   end
 
-  defp block_not_found(conn) do
+  defp to_json(attribute, module) when is_struct(attribute) do
```

Comment:
you probably want to move this function to a more generic module?

Reply:
Could it be, 'lib/beacon_api/utils'?